backend_test.go

```diff
@@ -347,8 +347,18 @@ func testDBBatch(t *testing.T, backend BackendType) {
 	require.NoError(t, batch.Set([]byte("c"), []byte{3}))
 	assertKeyValues(t, db, map[string][]byte{})
 
+	batchSize, err := batch.GetByteSize()
```

Comment:
Can we also add an assertion that byte size is 0 at the start, please? i.e. when batch is empty

Reply:
yesss

Reply:
there're some cases ( rockdb and pebble ) where byte size is not 0 for newly created batch, I think we should   skip this part.

Reply:
I see. Let's add your suggested assertion of 0 or smaller than 32. Let's please also add a comment with your findings

Reply:
```suggestion
	batchSize, err := batch.GetByteSize()
	require.Equal(t, db.Newbatch, batchsize)
```

this could be a workaround but redundant unless we are running a loop